README.md

```diff
@@ -147,13 +147,12 @@ If you are using a token in your project, you can execute `npm link '<token-name
 
 ### Publish
 
-Since every token is a `npm` package, changes to the token, even if the code is merged, must need a new version published to take effect.
+Publish is automatic upon a PR get succesfully merged to master. **Please don't do manual publishing, in case of problem, open an issue**
+The following rules are automatically identified in CI:
 
```

Comment:
I agree about a message in the readme to ensure nobody publishes, but perhaps, for everything else, I would link to the CONTRIBUTING's section? Perhaps with a link like "click here for more details..."?
What do you think

Reply:
Good idea!